### ISSUE TITLE
Support something like "ecosystem" or "supports" for Labs projects

### ISSUE THREAD

**Issue by danfowler** [okfn/okfn.github.com#438]
e.g. these tools all support Data Packages and are part of a "[Frictionless Data](http://frictionlessdata.io/)" ecosystem, so the website should highlight that.

- http://okfnlabs.org/projects/dpm/
- http://okfnlabs.org/projects/goodtables/
- http://okfnlabs.org/projects/mira/
- http://okfnlabs.org/projects/core-datasets/

**Comment by rgrp** [comment]
@danfowler good idea - and had similar thoughts about a `theme` category for projects in #402

**Comment by danfowler** [comment]
@rgrp cool.  Based on conversations last night in Barcelona, I think this is relatively high priority as an organizing principle for Labs projects.  If different projects could possibly build on each other and work together, that should be highlighted.

**Comment by rgrp** [comment]
@danfowler i agree - did you see the work in #402 in the spreadsheet? What can we do to get this rolled out to the site?
